lib/elixir/pages/Writing Documentation.md

```diff
@@ -51,6 +53,20 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
       size
     end
 
+## Documentation metadata
+
+Elixir allows developers to attach random metadata to the documentation. This is done by passing a keyword list to the relevant attribute (such as `@moduledoc`, `@typedoc`, and `@doc`). A commonly used metadata is `:since`, which annotates in which version that particular module, function, type, or callback was added on, as shown in the example above.
```

Comment:
or callback was added ~~on~~

Reply:
So @eksperimental suggested to add "on" and since I don't know which way is correct, I asked Chris and he said without "on" is correct. So I will remove it.